mlrun/api/crud/notifications.py

```diff
@@ -17,9 +17,26 @@
 import sqlalchemy.orm
 
 import mlrun.api.api.utils
+import mlrun.api.db.sqldb.db
+import mlrun.api.utils.scheduler
 import mlrun.api.utils.singletons.db
+import mlrun.api.utils.singletons.scheduler
+import mlrun.common.schemas
 import mlrun.utils.singleton
 
+set_notification_methods = {
+    "run": {
+        "factory": mlrun.api.utils.singletons.db.get_db,
+        "method_name": mlrun.api.db.sqldb.db.SQLDB.set_run_notifications.__name__,
+        "identifier_key": "uid",
+    },
+    "schedule": {
+        "factory": mlrun.api.utils.singletons.scheduler.get_scheduler,
+        "method_name": mlrun.api.utils.scheduler.Scheduler.set_schedule_notifications.__name__,
+        "identifier_key": "name",
+    },
+}
+
```

Comment:
why not moving it under `set_object_notifications` itself?
if become common (or commonly use, Id suggest create a factory)